README.md

```diff
@@ -65,17 +112,15 @@ Execute installation script
        - `mosip.certify.vciplugin.sunbird-rc.credential-type.{credential type}.static-value-map.issuerId`.
        - `mosip.certify.vciplugin.sunbird-rc.credential-type.{credential-type}.cred-schema-id`.
    - The `$.schema[0].author` DID goes to the config ending in issuerId and `$.schema[0].id` DID goes to the config ending in `cred-schema-id`.
-9. Once the Esignet and Certify properties are configured, proceed to select Certify from the option provided in the installation steps.
-10. The installation of Certify will encompass the following services:
+10. Once the Esignet and Certify properties are configured, proceed to select Certify from the option provided in the installation steps.
+11. The installation of Certify will encompass the following services:
     * [Esignet Service](https://github.com/mosip/esignet)
     * [Certify Service](https://github.com/mosip/inji-certify)
-11. Download the postman collection and environment for sunbird use case from [here](docker-compose/docker-compose-certify/postman-collections).
-12. Create Client from Create OIDC client API, add redirect uri 'http://localhost:3001'.
-13. Change `aud` variable in environment to 'http://localhost:8088/v1/esignet/oauth/v2/token' and set `audUrl` to http://localhost:8088
+12. Download the postman collection and environment for sunbird use case from [here](docker-compose/docker-compose-certify/postman-collections/sunbird).
+13. Create Client from Create OIDC client API, add redirect uri 'http://localhost:3001'.
 14. Perform a Knowledge based authentication(KBA) as specified in the Postman collection.
     * perform the authorize callback request
     * in the /authorization/authenticate request update the challenge to a URL-safe base64 encoded string with the KBA details such as `{"fullName":"Abhishek Gangwar","dob":"1967-10-24"}`, one can use an [online base64 encoding service](https://base64encode.org) for the same.
-    * in the /issuance/credential api inside pre-request script section change the aud env variable to  -> "aud" : pm.environment.get('audUrl')
```

Comment:
I think we should also have a section / note to point to the docker compose of inji-web and tell that can be used for a trying a end to end UI flow.

Reply:
add a note for inji web setup